p/python-hatchling/manifest.x86_64.jsonc

```diff
@@ -4,30 +4,25 @@
 	"packages": {
 		"python-hatchling": {
 			"build-depends": [
-				"binary(python3)",
-				"python-editables",
-				"python-installer",
-				"python-packaging",
-				"python-pathspec",
-				"python-pluggy",
-				"python-trove-classifiers",
-				"python-wheel"
+				"python(editables)",
+				"python(pathspec)",
+				"python(pluggy)",
+				"python(trove-classifiers)"
 			],
 			"depends": [
 				"binary(python3)",
-				"python(editables(3.11))",
```

Comment:
Does it still need editables as a builddep if it's no longer a rundep?

Reply:
nice catch